TESTS/mbed_platform/minimal-printf/compliance/README.md

```diff
@@ -0,0 +1,11 @@
+# Description
+
+This document describes how to run minimal-printf tests.
+
+## Running tests
+
+You can use the following command to run tests:
+
+`mbed test -m K64F -t GCC_ARM -n *printf* -v -c`
+
+Do not use --profile minimal-printf so minimal-printf is not compared with itself.
```

Comment:
```suggestion
<aside class="warning">
Do not use --profile minimal-printf so minimal-printf is not compared with itself.
</aside>
```

Reply:
This is not used by default so the current text (without emphasis) is fine.

Reply:
This is supported by `Slate` so you can ignore my comment.
https://github.com/lord/slate/wiki/Markdown-Syntax
Resolving